app/controllers/admin/budget_groups_controller.rb

```diff
@@ -58,7 +58,11 @@ def groups_index
     end
 
     def budget_group_params
-      valid_attributes = [:max_votable_headings]
+      valid_attributes = [
+        :max_votable_headings,
+        :number_votes_per_heading,
```

Comment:
How about `max_votes_per_heading`, I think it's a little more consistent with `max_votable_headings` or maybe `max_votes` if we move it to the `heading` table? 🤔